spec/core_functions/string/split.hrx

```diff
@@ -0,0 +1,179 @@
+<===> options.yml
+---
+:ignore_for:
+- libsass
+
+<===>
+================================================================================
+<===> empty/input.scss
+a {b: split("")}
```

Comment:
This function won't be defined globally, so this should load it from `sass:string`.

Reply:
These are all loading for `sass:string` now.